src/components/dropdownSearch/DropdownSearch.tsx

```diff
@@ -87,7 +102,8 @@ export class DropdownSearch extends React.Component<IDropdownSearchProps, void>
     )
       .map((option: IDropdownOption, index: number, options: IDropdownOption[]) => {
         const optionClasses = classNames({
```

Comment:
Personnellement je sortirais `this.props.selectedOptions.getFirstElement()`

`firstElement = this.props.selectedOptions.getFirstElement();`

`'state-selected': firstElement && option.value === firstElement.value`

Rendu là ça coute pas plus chère de switcher l'ordre:

`'state-selected': firstElement && firstElement.value === option.value`

C'est de petits détails mais je trouve qu'on comprends plus facilement la logique ainsi